src/main/CommandHandler.cpp

```diff
@@ -437,7 +438,7 @@ CommandHandler::info(std::string const& params, std::string& retStr)
 {
     Json::Value root;
 
-    LedgerManager& lm = mApp.getLedgerManager();
+    auto& lm = mApp.getLedgerManager();
```

Comment:
Without real reason, I was just changing code around it. I'll revert it back and rebase so this commit will not have this random change.